### ISSUE TITLE
Fail to start a project on Windows 10

### ISSUE THREAD

**Issue by AleffSouza** [Hashnode/mern-cli#13]
I just tried the fist steps to create an application given in the project's website and when I run `npm install` I got some warns:
```
npm WARN babel-plugin-webpack-loaders@0.5.0 requires a peer of webpack@>=1.12.9 <3.0.0 but none was installed.
npm WARN chunk-manifest-webpack-plugin@0.1.0 requires a peer of webpack@^1.4.0-beta6 but none was installed.
npm WARN extract-text-webpack-plugin@1.0.1 requires a peer of webpack@^1.9.11 but none was installed.
```

After that I tried `npm start`:

```
Error: C:/.../my_app/client/modules/App/App.js: Command failed: node C\:\\...\\my_app\\node_modules\\webpack\\bin\\webpack.js C\:\\...\\my_app\\client\\modules\\App\\App.css C\:\\...\\AppData\\Local\\Temp\\.webpack.res.1466572316905_525303.js --config ./webpack.config.babel.js --bail
module.js:442
    throw err;
    ^

Error: Cannot find module 'C\:\...\my_app\node_modules\webpack\bin\webpack.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:449:3

    at checkExecSyncError (child_process.js:475:13)
    at execSync (child_process.js:515:13)
    at exports.default (C:\...\my_app\node_modules\babel-plugin-webpack-loaders\lib\runWebPackSync.js:41:51)
    at PluginPass.CallExpression (C:\...\my_app\node_modules\babel-plugin-webpack-loaders\lib\plugin.js:93:60)
    at newFn (C:\...\my_app\node_modules\babel-traverse\lib\visitors.js:301:19)
    at NodePath._call (C:\...\my_app\node_modules\babel-traverse\lib\path\context.js:76:18)
    at NodePath.call (C:\...\my_app\node_modules\babel-traverse\lib\path\context.js:48:17)
    at NodePath.visit (C:\...\my_app\node_modules\babel-traverse\lib\path\context.js:106:12)
    at TraversalContext.visitQueue (C:\...\my_app\node_modules\babel-traverse\lib\context.js:167:16)
    at TraversalContext.visitSingle (C:\...\my_app\node_modules\babel-traverse\lib\context.js:118:19)
    at TraversalContext.visit (C:\...\my_app\node_modules\babel-traverse\lib\context.js:211:19)
    at Function.traverse.node (C:\...\my_app\node_modules\babel-traverse\lib\index.js:161:17)
    at NodePath.visit (C:\...\my_app\node_modules\babel-traverse\lib\path\context.js:116:19)
    at TraversalContext.visitQueue (C:\...\my_app\node_modules\babel-traverse\lib\context.js:167:16)
    at TraversalContext.visitMultiple (C:\...\my_app\node_modules\babel-traverse\lib\context.js:113:17)
    at TraversalContext.visit (C:\...\my_app\node_modules\babel-traverse\lib\context.js:209:19)
    at Function.traverse.node (C:\...\my_app\node_modules\babel-traverse\lib\index.js:161:17)

```

The same happens cloning the mern-starter project.

Node 6.2.0
npm 3.8.9
MongoDB 2.6.6

**Comment by somus** [comment]
There is a problem with `babel-plugin-webpack-loaders` in windows. Refer Hashnode/mern-starter#170 for more info. @sosana has opened a PR with `babel-plugin-webpack-loaders` to fix that. Will be fixed once that PR is accepted.

**Comment by somus** [comment]
Fixed on Hashnode/mern-starter#176. Pull again and check.
